File: AnalisisNumerico-master/Problemas R/EvaluarPolinomio.R

polinomio = function(x){
  coeficientes <- c(2, 0, -3, 3, 4)
  resultado = 0
  for (i in 0:length(coeficientes)) {
    resultado = resultado * x + coeficientes[i]
    }
  return(resultado)
}
result = polinomio(-2)
cat("La función da: ", result)

File: AnalisisNumerico-master/Problemas Python/EvaluarPolinomio.py
#Despues de buscar, se decidió usar el Método de Horner, el cual la profesora ya habia mencionado

def polinomio(x):
        coeficientes=[-4, 3, -3, 0, 2]
        resultado = 0 
        for i in reversed(range(len(coeficientes))):
                resultado = coeficientes[i] + resultado * x
        return resultado
print(polinomio(-2))